install/streams.csv

```diff
@@ -1229,10 +1229,9 @@ MRZ,10,1,false,false,false,2003-07-31T00:00:00Z,9999-01-01T00:00:00Z
 MRZ,10,50,false,false,true,1992-04-08T04:21:00Z,2003-04-15T01:43:00Z
 MRZ,10,100,false,false,false,2003-07-31T00:00:00Z,9999-01-01T00:00:00Z
 MRZ,20,1,false,false,false,2003-07-31T00:00:00Z,2011-04-13T00:00:00Z
-MRZ,20,50,false,false,false,2011-04-13T00:00:02Z,2019-07-30T22:50:00Z
-MRZ,20,200,false,false,true,2003-07-31T00:00:00Z,2019-07-30T22:50:00Z
-MRZ,20,200,false,false,false,2019-08-01T04:30:00Z,2019-10-31T23:01:00Z
-MRZ,21,200,false,false,false,2019-10-31T23:02:00Z,9999-01-01T00:00:00Z
+MRZ,21,50,false,false,false,2011-04-13T00:00:02Z,2019-07-30T22:50:00Z
+MRZ,21,200,false,false,true,2003-07-31T00:00:00Z,2019-07-30T22:50:00Z
```

Comment:
if the opening date/time of LC 21 in `sites.csv` is 2011-04-13/00:00:02, should the start of this stream be the same with that LC 21 opening?

Reply:
yeap will do :0